src/commands/dune

```diff
@@ -0,0 +1,14 @@
+(library
+  (name merlin_commands)
+  (public_name merlin-lib-commands)
```

Comment:
```
(public_name merlin-lib.commands)
```
If it does not actually introduce new dependencies, we might simply ship it as a part of `merlin-lib`

Reply:
Done in the last commit https://github.com/ocaml/merlin/pull/1758/commits/24e97948e1d3f3774484f26e6c18338ac266d02a